scripts/Interface/reduction_gui/reduction/scripter.py

```diff
@@ -314,10 +319,10 @@ def __init__(self, name="", facility=""):
         if HAS_MANTID:
             config = ConfigService.Instance()
             try:
-                head, tail = os.path.split(config.getUserFilename())
+                head, _tail = os.path.split(config.getUserFilename())
                 if os.path.isdir(head):
                     self._output_directory = head
-            except:
+            except (StopIteration, StandardError, Warning):
```

Comment:
Are these all exception types which can occur?  Does pycharm display all possible exception types? Would be really usefull for me to know that.